tensorrec/tensorrec.py

```diff
@@ -98,13 +101,17 @@ def __init__(self, n_components=100,
         # A map of every graph hook attr name to the node name after construction
         # Tensors and operations are stored separated because they are handled differently by TensorFlow
         self.graph_tensor_hook_node_names = {}
+        self.biased_graph_tensor_hook_node_names = {}
         self.graph_operation_hook_node_names = {}
 
     def _clear_graph_hook_attrs(self):
         for graph_tensor_hook_attr_name in self.graph_tensor_hook_attr_names:
             self.__setattr__(graph_tensor_hook_attr_name, None)
         for graph_operation_hook_attr_name in self.graph_operation_hook_attr_names:
             self.__setattr__(graph_operation_hook_attr_name, None)
+        if self.biased:
```

Comment:
All these will be able to go away if the bias hooks are conditionally added to the right list.